SimG4CMS/Calo/src/CaloSD.cc

```diff
@@ -321,7 +321,7 @@ bool CaloSD::checkHit() {
       found = true;
     }
   } else if (nCheckedHits > 0) {
-    int minhit = (theHC->entries() > nCheckedHits ? theHC->entries() - nCheckedHits : 0);
+    size_t minhit = (theHC->entries() > nCheckedHits ? theHC->entries() - nCheckedHits : 0);
```

Comment:
@mrodozov , I would move everything to "int":
int nhits = theHC->entries();
int minhit = std::max(nhits - nCheckedHits, 0);

the same everywhere - easy to read and no mix of types anymore.